var/spack/repos/builtin/packages/bml/package.py

```diff
@@ -16,6 +16,7 @@ class Bml(CMakePackage):
     git      = "https://github.com/lanl/bml.git"
 
     version('develop', branch='master')
+    version('1.3.1', sha256='75bbbf8e224e7707ee930918985ef78d369fc616eb67012b43a4d943648306d0')
```

Comment:
After retagging, the new hash is `17145eda96aa5e550dcbff1ee7ce62b45723af8210b1ab70c5975ec792fa3d13`.

Reply:
Updated.